__tests__/api-endpoints.test.js

```diff
@@ -190,3 +190,117 @@ describe("/api/articles/:article_id/comments - sad path", () => {
   })
 });
 
+describe("POST - /api/articles/:article_id/comments", () => {
+  test("responds with the posted comment, an object with username & body key / values", () => {
+    const commentObj = {
+      username: "lurker",
+      body: "might be the best api i have ever seen in my life",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(commentObj)
+      .expect(201)
+      .then(({body: comment} ) => {
+        //console.log(commebody.nt.body)
+        expect(comment.postedComment[0]).toBeObject();
+        expect(comment.postedComment[0].author).toBe("lurker");
+        expect(comment.postedComment[0].body).toBe(
+          "might be the best api i have ever seen in my life"
+        );
+      });
+  });
+  test("responds with a single comment response only", () => {
+    const commentObj = {
+      username: "lurker",
+      body: "this is a comment somehow",
+    };
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(commentObj)
+      .expect(201)
+      .expect(({body: comment}) => {
+        expect(comment.postedComment.length).toBe(1);
+      });
+  });
+  test("discounts extra data provided for comment insertion", () => {
+    const commentObj = {
+      crisps: "yummy",
+      username: "lurker",
+      body: "this is a comment somehow",
+      birds: "are lovley"
+    };
+    return request(app)
+    .post("/api/articles/3/comments")
+    .send(commentObj)
+    .expect(201)
+    .expect(({body: comment}) => {
+      expect(Object.keys(comment.postedComment[0])).toEqual(["author", "body"])
+  })})
+});
+
+describe("POST - /api/articles/:article_id/comments - sad path", () => {
```

Comment:
I like that you divided out the happy and sad paths into different describes, it might even look neater if the describe blocks were nested